.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy Release
+
+on:
+  # Trigger the workflow on push action in develop branch
+  # So it will trigger when the PR of the feature branch was merged.
+  push:
+    branches:
+      - main
+jobs:
+  app_deploy:
+    name: Deploy and upload to JitPack
+    runs-on: ubuntu-latest
+
+    steps:
```

Comment:
FYI: We are having a convention about formatting for Github Actions yml file on [Compass](https://nimblehq.co/compass/development/code-conventions/github-actions/#formatting). We would update if we have spare time, otherwise, we can leave it as it is.

Reply:
@hoangnguyen92dn No problem, I updated it.